### PR TITLE
chore: upgrade charts to Flux v2.3.0 release

### PR DESCRIPTION
**Pull request by seaneagan**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [ ] [DCO](https://github.com/fluxcd-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [ ] Chart Version bumped
- [ ] helm-docs are updated
- [ ] Helm chart is tested
- [ ] Update artifacthub.io/changes in Chart.yaml
- [ ] Run `make reviewable`

### CODE REVIEW

**Comment by stefanprodan** [comment]
Hey @seaneagan please signoff your commit

**Comment by seaneagan** [comment]
@stefanprodan added the missing minor chart version bumps as well

File: charts/flux2-sync/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: flux2-sync
 type: application
 version: 1.8.2
```
Comment (stefanprodan) [inline charts/flux2-sync/Chart.yaml:4]: Please bump this to 1.9.0

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @seaneagan 🏅

**Pull request merged**